portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -54,14 +54,53 @@ definition {
 	}
 
 	@description = "Verify that a Color Field can be duplicated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeDuplicated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133135 CanBeDuplicated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		FormFields.editHelpTextField(fieldHelpText = "Help Text Test");
```

Comment:
hi, @piresdm ,Can you use FormViewBuilder.setFieldHelpText instead of you write the new one? thanks

Reply:
Hi, @chaowang-123!
I guess i can`t do that, because @natocesarrego asked us to not use macros inside the FormViewBuilder. 
Could you check this information?

Reply:
hi, @piresdm , yes, you are right, I check the information just now